userspace/libsinsp/test/events_proc.ut.cpp

```diff
@@ -396,7 +397,7 @@ TEST_F(sinsp_with_test_input, spawn_process)
 	ASSERT_EQ(get_field_as_string(evt, "proc.exeline"), "/bin/test-exe -c 'echo aGVsbG8K | base64 -d'");
 	ASSERT_EQ(get_field_as_string(evt, "proc.tty"), "34818");
 	ASSERT_EQ(get_field_as_string(evt, "proc.vpgid"), "1");
-	ASSERT_EQ(get_field_as_string(evt, "user.loginuid"), "1000");
+	ASSERT_EQ(get_field_as_string(evt, "user.loginuid"), "4294967294");
```

Comment:
that's fine! we could also add just a simple test that generates a process with a clone+execve with a `loginuid==UINT32_MAX` and check if it is rendered as the "-1" string, WDYT?

Reply:
:woman_shrugging: the new new that came out ...

Reply:
Great idea, done.